aether-odk-module/aether/odk/api/xform_utils.py

```diff
@@ -297,6 +299,7 @@ def parse_xform_to_avro_schema(
         current_name = xpath.split('/')[-1]
         current_doc = definition.get('label')
         current_choices = definition.get('choices')
+        current_extended_property = definition.get(VISUALIZATION_KEY)
```

Comment:
Why not `current_visualizer`? (simpler is better)

```python
        current_visualizer = definition.get('visualizer')
```

You only need the `VISUALIZATION_KEY` for the resultant AVRO schema, internally you can use simpler and easier names.